pkgs/by-name/le/legends-of-equestria/package.nix

```
@@ -0,0 +1,141 @@
+{
+  lib,
+  stdenv,
+  runCommand,
+  megacmd,
+  unzip,
+  makeWrapper,
+  autoPatchelfHook,
+  makeDesktopItem,
+  copyDesktopItems,
+  libgcc,
+  dbus,
+  xorg_sys_opengl,
+  systemd,
+  libcap,
+  libdrm,
+  pulseaudio,
+  libsndfile,
+  flac,
+  libvorbis,
+  libopus,
+  mpg123,
+  lame,
+  libGL,
+  vulkan-loader,
+  libasyncns,
+  xorg,
+}:
+
+let
+  pname = "legends-of-equestria";
+  version = "2024.05.01";
+  description = "Free-to-play MMORPG";
+  runtimeDeps =
+    [
+      dbus.lib
+      xorg_sys_opengl
+      systemd
+      libcap.lib
+      libdrm
+      pulseaudio
+      libsndfile
+      flac
+      libvorbis
+      mpg123
+      lame.lib
+      libGL
+      vulkan-loader
+      libasyncns
+    ]
+    ++ (with xorg; [
+      libX11
+      libxcb
+      libXau
+      libXdmcp
+      libXext
+      libXcursor
+      libXrender
+      libXfixes
+      libXinerama
+      libXi
+      libXrandr
+      libXScrnSaver
+    ]);
+in
+stdenv.mkDerivation {
+  inherit pname version;
+  src =
+    runCommand "mega-loe"
+      {
+        inherit pname version;
+        nativeBuildInputs = [
+          megacmd
+          unzip
+        ];
+        url = "https://mega.nz/file/NjIwwJoK#MVi3C3FAcSQPd7FRpQc0CoStBG8jSFuPn0jD-pG3zY0";
+        outputHashAlgo = "sha256";
+        outputHash = "VXBtEB3G5MTrWn9OOvmCG3sDoasjbKkLJruhdQZa4SQ=";
+        outputHashMode = "recursive";
+      }
+      ''
+        export HOME=$(mktemp -d)
+        dest=$HOME/mega-loe
+        mkdir -p $dest
+        mega-get "$url" $dest
+        mkdir -p $out
+        unzip -d $out $dest/$(ls $dest)
```

```suggestion
        unzip -d $out $dest/*.zip
```
unzip would only unpack zip files, right?

And I think https://mywiki.wooledge.org/ParsingLs applies here.